USER-INTERFACE-INTERFACE.md

```diff
@@ -359,8 +359,9 @@ during creation of the database, it never changes. It's basically a read-only va
 
 * `clandestinePort`: The port on which the Node is currently listening for connections from other Nodes.
 
-* `consumingWalletDerivationPathOpt`: This is the derivation path (from the mnemonic seed) of the consuming wallet.
-  More than likely, it's m/44'/60'/0'/0/0.
+* `consumingWalletPrivateKey`: This is the private key of the consuming wallet, as a 64-digit hexadecimal number.
```

Comment:
It may pay to add text to explain this is a secret value, so won't be displayed without the db-password supplied

@dnwiebe